### DISCUSSION TITLE
I want to receive license key again but I couldn't

### DISCUSSION THREAD

**Discussion by nikin78** [aclap-dev/video-downloadhelper#459] [Help and Q&A] [answered]
### Your question

I used your automated system to receive the key again from my email, but there was no email come even on spam box.
I also tried to get your help by sending email on support@downloadhelper.net but your email rejected my email.
What else should I do now?
My email address is prof.patpn@gmail.com

**Comment by jcfrog** [comment, marked as answer]
Hi,
I sent you an email.
